
teach_motions:
  output_tf_frame:  base_link  # Frame to save the recorded data in
  interpolation_timestep:  0.008  # 125 Hz
  num_arms:  1
  joints_topic:  joint_states
  teach_mode_velocity_scale:  0.25  # Scale the speed of taught motions. Affects the recorded times only.
  ee0:
    movegroup_name:  right_ur5
    ee_frame_name: right_ur5_ee_link
    jog_cmd_topic:  jog_arm_server/right_arm/delta_jog_cmds

  # ee0:
  #   movegroup_name:  left_ur5
  #   ee_frame_name: left_ur5_ee_link
  #   jog_cmd_topic:  jog_arm_server/left_arm/delta_jog_cmds
  # ee1:
  #   movegroup_name:  right_ur5
  #   ee_frame_name: right_ur5_ee_link
  #   jog_cmd_topic:  jog_arm_server/right_arm/delta_jog_cmds

compliant_replay:
  ee0:
    force_torque_topic:  right_ur5_wrench
    force_torque_frame:  right_ur5_base

  # ee0:
  #   force_torque_topic:  left_ur5_wrench
  #   force_torque_frame:  left_ur5_base
  # ee1:
  #   force_torque_topic:  right_ur5_wrench
  #   force_torque_frame:  right_ur5_base